### DISCUSSION TITLE
Issues with Tailwind Preset and unstyled mode

### DISCUSSION THREAD

**Discussion by carlosdelros** [primefaces/community#1585] [PrimeVue]
Hi! 

According to the docs, in order to use tailwind preset we need to set the unstyled property to true 
app.use(PrimeVue, {
    unstyled: true
});

However, when using certain components the behavior is inconsistent. For example taking a look at the DataTable component with the resizableColumns functionality:
(Without Tailwind preset, unstyled: false) https://primevue.org/datatable/#column_resize
(With Tailwind preset, unstyled: true) https://tailwind.primevue.org/datatable/#column_resize

As you can see the tailwind preset version has at least 2 bugs: 
1. when resizing the columns the surrounding text is selected
2. when collapsing the column the overflow of the cell is not hidden, it bleeds to the adjacent cell.

Looking at the [source code](https://github.com/primefaces/primevue/blob/1a68c7144e3b648643e5e91491a11a3a349b7a66/components/lib/datatable/DataTable.vue#L1238)  you can see that some of this behavior happens because certain behavior is not enabled if the unstyled property is set to true; in this case the class 'p-unselectable-text' is not added because we are working with the unstyled: true. Also searching for this condition (!this.isUnstyled)  throughout the source code shows that this check is done in multiple places and multiple components which probably means that there are bugs in those components when using the tailwind preset. 

- Why are we not enabling some of the behaviors when in unstyled mode?
- Am i missing something that I should be doing in order to have the correct behavior?
- I know I could manually mitigate these issues, but it's not ideal. I know example bug #2 would be simple to fix, but #1 not so much.

Thoughts?

**Comment by drbarto** [comment]
We have similar issues with the unstyled/Tailwind variants. Mostly the components work as the styled ones do, but the last 5% are not there yet. I wonder if the PrimeVue team considers this feature complete, or still work-in-progress? Would be great if someone could point out any communication regarding this topic.
